lib/fc.js

```diff
@@ -402,22 +445,76 @@ async function backupTemplateFile(tplPath) {
   console.log(green(`\nFun automatically backups the original ${path.basename(tplPath)} file to ${newPath}`));
 }
 
-const NODE_RUNTIME = ['nodejs6', 'nodejs8', 'nodejs10'];
+function ensureMountTargetsExist(mountTargets) {
+  if (_.isEmpty(mountTargets)) {
+    throw new Error(red('Nas has not configured the mount point yet, please go to the console https://nas.console.aliyun.com/ to manually create the mount point.'));
+  }
+}
 
-async function nasAutomationConfigurationIfNecessary({
-  nasFunctionName,
+function ensureSecurityGroupsExist(securityGroups) {
+  if (_.isEmpty(securityGroups)) {
+    throw new Error(red('The security group has not been configured, please refer to https://help.aliyun.com/knowledge_detail/72959.html#config'));
```

Comment:
\nThere is no SecurityGroup available. You need to login to the vpc console https://ecs.console.aliyun.com/ to create one and then use 'fun deploy' to deploy your resources again.

Reply:
done